### ISSUE TITLE
Baked books: Content in Section 14.5 in Physics is overlapping with image in reference view

### ISSUE THREAD

**Issue by Malar-Natarajan** [openstax/tutor#185]
**Describe the bug**
Baked books: Content in Section 14.5 in Physics is overlapping with image in reference view
See screenshot:

https://drive.google.com/file/d/1KDv0JusYvB2jTSUDWdpp8KIL17X47VVE/view?usp=sharing

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by carlycurran** [comment]
@brittany-johnson will this be fixed with your new image float and padding rules?

**Comment by brittany-johnson** [comment]
@carlycurran Yes! The text won't overlap with the image, but the image is pretty big. Since the content is at least readable now, I think this is a band-aid fix. This is what it will look like once the code is pushed to master: 

![screen shot 2018-12-20 at 12 07 04 am](https://user-images.githubusercontent.com/24909007/50268263-b9283d80-03ef-11e9-89c4-4fe6ed8aaf67.png)
![screen shot 2018-12-20 at 12 07 12 am](https://user-images.githubusercontent.com/24909007/50268264-b9283d80-03ef-11e9-89be-25b8d895f091.png)


The other day, we did discuss applying the max width of 50% to the images, but I found that it doesn't work very well for all images because they are imported at such varying sizes and smaller images become almost impossible to read. I think I have a soulution to this for horizontal images, but not vertical images yet. 

All of this ^^^ is in this PR https://github.com/openstax/tutor-js/pull/2455#issuecomment-448877877
